server/testutils/factories/cas1SpaceBookingDeparture.ts

```diff
@@ -0,0 +1,23 @@
+import { Factory } from 'fishery'
+import { faker } from '@faker-js/faker'
+import { type Cas1SpaceBookingDeparture } from '@approved-premises/api'
+
+class Cas1SpaceBookingDepartureFactory extends Factory<Cas1SpaceBookingDeparture> {}
+
+export default Cas1SpaceBookingDepartureFactory.define(() => {
+  return {
+    notes: faker.word.words(20),
+    parentReason: {
+      id: faker.string.uuid(),
+      name: `${faker.word.noun()} ${faker.word.verb()}`,
+    },
+    reason: {
+      id: faker.string.uuid(),
+      name: `${faker.word.noun()} ${faker.word.verb()}`,
+    },
+    moveOnCategory: {
+      id: faker.string.uuid(),
+      name: `${faker.word.noun()} ${faker.word.verb()}`,
+    },
```

Comment:
These are all `referenceData`, could we use the factories here? Watch out for the specific one for `DepartureReason`, which adds the `parentReasonId`.